resultbrowser/src/js/resultbrowser.js

```diff
@@ -547,7 +547,7 @@
 
     function isJsonContent(contentType) {
         //  e.g. "application/json" or "application/<...>+json"
-        return /^application\/(.+\+)?json$/.test(contentType);
+        return /^application\/(.+\+)?json/.test(contentType);
```

Comment:
This change might fix the issue but introduces another one as we will treat content as JSON that might not be JSON, e.g. [application/jsonpath](https://www.iana.org/assignments/media-types/application/jsonpath).

Or in other words: the regex was too strict before and is too lax now :smile: 

What about to keep the regex but sanitize the input instead?
```suggestion
        contentType = (contentType || "").split(";")[0].trim();
        return /^application\/(.+\+)?json$/.test(contentType);
```